### ISSUE TITLE
receipt + nfc card + discount

### ISSUE THREAD

**Issue by darkban** [karask/bitcoinpos#6]
it would be nice to implement 3 very important functions

1. payment receipt sent by email or if you have an external or integrated printer print it directly
2. implement the nfc function
3. add a button where you can apply a discount on the cart for example 5% or 10% or manual.

**Comment by karask** [comment]
Good suggestions. There is a pipeline with features to be added so not sure when we'll have time to work on them.

Currently, an option to work on testnet is being worked on so that a user can test the app with fake money if he wishes. Then, there is an option to accept other coins as well. Maybe after that we could look into NFC.

Regarding the email, how do you expect that to work since it will not be practical to ask the emails of customers. Do you mean to sent an email to the merchant?

**Comment by darkban** [comment]
It is my intention to send an email to the customer's email where it certifies the successful transition ... and you could also add the receipt print directly from the application for external or integrated printers.

**Comment by karask** [comment]
OK. How would you collect the customer's email? Will you ask them before the purchase? I don't understand the practical aspect of the suggestion. Note that there is no login process to acquire emails beforehand.

Regarding external printers, yes, that was asked beforehand and it will be scheduled some time.

**Comment by darkban** [comment]
Yes, correct, it will ask the customer and will serve those who do not have a thermal printer and will be used to track the customer because in this way if the customer pays does not have any receipt certifying it ...
obviously in the settings in addition to the name of the store you will have to add the address, city, province, email, website, and any government taxes that can also be printed on the email receipt or thermal printer.
